requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-docker-py>=0.3.2
+docker-py>=1.7.0
```

Comment:
I was testing against engine 1.10 rc1 and it broke on my laptop so I pushed the `requirements.txt` change in a hurry and went away; was talking about these broken runs:

https://travis-ci.org/d11wtq/dockerpty/jobs/104060547
https://travis-ci.org/d11wtq/dockerpty/jobs/104052086
https://travis-ci.org/d11wtq/dockerpty/jobs/104084403 

I have a thought how to improve the suite so it's easier to debug.